src/components/icons/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+export const EvergreenLogo = () => (
+  <svg
```

Comment:
You should run this through [svgomg](https://jakearchibald.github.io/svgomg/). Get a 50% reduction in the svg size and remove allot of the unnecessary svg filler

Reply:
good idea! totally forgot